region_parse.py

```diff
@@ -971,11 +976,84 @@ def update(self, event_key, window, amiibo, value):
         if amiibo is not None:
             self.set_value_in_bin(amiibo, value)
 
-    def get_template_signature(self):
+    def get_signature(self):
         """
         Generates signature of this section used to define it in regions
 
         :return: signature string
         """
-        parent = super().get_template_signature()
+        parent = super().get_signature()
         return "Text-" + parent
+
+
+class ImplicitSum:
+    """
+    Responsible for dealing with implicitly encoded values like jab/nair
+    """
+    def __init__(self, name, description, counterparts):
+        """
+        Initializes section
+
+        :param str name: name as str
+        :param str description: description
+        :param Tuple(str) counterparts: tuple of corresponding section signatures
+        """
+        self.name = name
+        self.description = description
+        self.primary_input_key = name
+        self.counterparts = counterparts
+
+    def get_counterpart_signatures(self):
+        """
+        gets all counterpart signatures to sum
+
+        :return: list of str of corresponding signatures
+        """
+        return self.counterparts
+
+    def get_widget(self, key_index):
+        """
+        Creates widget to be displayed in GUI
+
+        :param int key_index: key index to be used for input fields
+        :return: sg widget element
+        """
+        # doesn't call super() and doesn't set_primary key, updates are handled via DiffSumManager
+        # noinspection PyTypeChecker
+        return [[sg.Text(self.name + ":", font=("Arial", 10, "bold")), sg.Input(default_text=100, disabled=True, key=self.primary_input_key, size=(15, None))],
```

Comment:
add a `text_color = "black"` to the sg.Input so that it doesnt break on some themes.